salesforce/client.bal

```diff
@@ -245,6 +245,71 @@ public isolated client class Client {
         return check self.salesforceClient->delete(path);
     }
 
+    # Lists reports.
+    #
+    # + return - Array of Report if successful or else `error`
+    @display {label: "List Reports"}
+    isolated remote function listReports() returns @display {label: "Reports"} Report[]|error {
+        string path = utils:prepareUrl([API_BASE_PATH, ANALYTICS, REPORTS]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Deletes a report.
+    #
+    # + reportId - Report Id
+    # + return - `()` if the report deletion is successful or else an error
+    @display {label: "Delete Report"}
+    isolated remote function deleteReport(@display {label: "Report Id"} string reportId) returns error? {
+        string path = utils:prepareUrl([API_BASE_PATH, ANALYTICS, REPORTS, reportId]);
+        return check self.salesforceClient->delete(path);
+    }
+
+    # Runs an instance of a report synchronously.
+    #
+    # + reportId - Report Id
+    # + return - ReportInstanceResult if successful or else `error`
+    @display {label: "Run Report Synchronously"}
+    isolated remote function runReportSync(@display {label: "Report Id"} string reportId)
+            returns @display {label: "Report Instance Result"} ReportInstanceResult|error {
+        string path = utils:prepareUrl([API_BASE_PATH, ANALYTICS, REPORTS, reportId]);
+        return check self.salesforceClient->get(path);
+    }
+
+    # Runs an instance of a report asynchronously.
+    #
+    # + reportId - Report Id
+    # + return - ReportInstance if successful or else `error`
+    @display {label: "Run Report Asynchronously"}
+    isolated remote function runReportAsync(@display {label: "Report Id"} string reportId)
+            returns @display {label: "Report Instances"} ReportInstance|error {
```

Comment:
@display {label: "Report Instance"}

Instead of

@display {label: "Report Instances"}

Reply:
Done